### PR TITLE
Remove and update redundant keycode titles

### PR DESCRIPTION
**Pull request by noroadsleft**

### CODE REVIEW

File: configurator/src/store/modules/keycodes/app-media-mouse.js [hunk 1]
```diff
@@ -42,49 +41,21 @@ export default [
...
-  {
-    name: 'Mouse Right',
-    code: 'KC_MS_R',
-    title: 'KC_MS_R - Mouse Cursor Right'
-  },
+  { name: 'Mouse Up', code: 'KC_MS_U', title: 'Mouse Cursor Up' },
+  { name: 'Mouse Down', code: 'KC_MS_D', title: 'Mouse Cursor Down' },
+  { name: 'Mouse Left', code: 'KC_MS_L', title: 'Mouse Cursor Left' },
+  { name: 'Mouse Right', code: 'KC_MS_R', title: 'Mouse Cursor Right' },
   { width: 250 },
```
Comment (yanfali) [inline configurator/src/store/modules/keycodes/app-media-mouse.js]: Not sure these should change. It's either or not both. What do you think?

Comment (noroadsleft) [reply configurator/src/store/modules/keycodes/app-media-mouse.js]: I'm not sure I understand what you're getting at.

Comment (yanfali) [reply configurator/src/store/modules/keycodes/app-media-mouse.js]: Have you tested it? You should see the issue

Comment (noroadsleft) [reply configurator/src/store/modules/keycodes/app-media-mouse.js]: I mean, it shows a tooltip of both the keycode and the title on the tabs, and the keycode only if it's in the keymap (which I don't remember seeing until today). Is that what you're getting at?

Comment (yanfali) [reply configurator/src/store/modules/keycodes/app-media-mouse.js]: I'm kind of surprised. It's either or. Either show the code or the title. I think it's a good idea to show both though and I may make that change.

**Comment by yanfali** [comment]
awesome, thanks for this. Merged with commit 0e0ca8d3beb89c04db6f742a89eb59c426d72f6f

**Pull request closed without merging**